spec/models/cat_spec.rb

```diff
@@ -1,5 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "should validate name" do
+    cat = Cat.create
```

Comment:
What cat are you creating?  There will need to be attributes provided each time you create a cat.